crates/utils/src/brush.rs

```diff
@@ -1,67 +1,76 @@
 use crate::prelude::*;
 
-/// A `Brush` describes how a shape is filled or stroked.
-/// A `Brush` can be parse from a string(as that is how the brushes are parsed in the themes)
-/// For example by an expression you have three methods to define a solid color:
+/// A `Brush` describes how a shape is filled or stroked.  A `Brush`
+/// can be parse from a string (themes are parsing brushes that way).
+/// With the given implementation you can choose between three methods
+/// to define a solid color:
 ///
-/// A. Use color codes, for example `#f00` will give you red or if you write #00ff00 you will got blue
-/// also you can use alpha.
+/// A. Use `color codes`
 ///
-/// B. Use a function, currently the unique available functions to interpret a color are `rgb`, `hsv`, `hsb`
-/// (which is the same as `hsv`), `hsl` and its respective alpha variants(rgba by example). So you can define
-/// a color as `hsl(197, 71%, 73%)` and you will get a pretty skyblue color. For `rgb` and `rgba` the
-/// range of the values are 0-255 but from all others the values are as follows: hsva(0.0-360.0, 0.0-1.0, 0.0-1.0, 0.0-1.0)
-/// but the % sign can change that, it can be added to individual parameters and change its range depending on the function,
-/// for `rgb` and `rgba` you can write an % after a parameter and it will be mapped from 0.0-100.0, or if the number that you write
-/// is between 0.0 and 1.0 you value will be mapped from that range. But if you function is not rgb or rgba, you can not use a
-/// percent sign in the first parameters but in the following parameters the sign will make OrbTk interpret it in the 0.0-100.0 range.
+/// Define the value with a symbol "#" followed by 6 letters or numbers.
```

Comment:
the lengths are 3, 6 and 4, 8 for non-alpha and alpha colors respectively. I prefer be clear and verbose